tap_iterable/streams.py

```diff
@@ -147,7 +147,13 @@ def sync_data_export(self, state):
                             rec["transactionalData"] = json.loads(rec["transactionalData"])
                         except KeyError:
                             pass
-                        self.update_session_bookmark(rec[self.replication_key])
+                        try:
+                            self.update_session_bookmark(rec[self.replication_key])
```

Comment:
What about `rec.get(self.replication_key, request_end_date)` to avoid the try/except? Do you think the logging will be helpful?

Reply:
I like your suggestion. The try/except is just debugging cruft and the logs are noisy anyway